### PR TITLE
add react

### PR DESCRIPTION
**Pull request by gdhaison**
===> Title: React

===> Redmine ticket: https://edu-redmine.sun-asterisk.vn/issues/22620

===> Attachment files:

### CODE REVIEW

**Comment by gdhaison** [comment]
ready

File: app/assets/javascripts/components/branch.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      url: finalUrl,
+    })
+  }
+
+  render() {
+    var branchs = this.props.branchs.map((branch) => {
+      return (
+        <tr>
+          <td>{branch.name}</td>
+          <td><a className="btn btn-primary" href="#">Update</a></td>
```
Comment (ngocvt-0484) [inline app/assets/javascripts/components/branch.jsx]: sd i18n trong js đi e

Comment (gdhaison) [reply app/assets/javascripts/components/branch.jsx]: vâng ạ

**Comment by gdhaison** [comment]
ready

**Pull request merged**